docs/docs/03-interactivity-and-dynamic-uis.md

```diff
@@ -11,22 +11,25 @@ You've already [learned how to display data](/react/docs/displaying-data.html) w
 ## A Simple Example
 
 ```javascript
-var LikeButton = React.createClass({
-  getInitialState: function() {
-    return {liked: false};
-  },
-  handleClick: function(event) {
+class LikeButton extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      liked: false
+    }
+  }
+  handleClick = () => {
```

Comment:
We won’t use class properties in the docs. We only want to use stable ES2015 syntax. So if we need to bind an event handler, we will do this in the constructor.